packages/contentstack/src/hooks/prerun/latest-version-warning.ts

```diff
@@ -0,0 +1,60 @@
+import { cliux, configHandler, HttpClient, LoggerService } from '@contentstack/cli-utilities';
+import * as semver from 'semver';
+interface ICacheData {
+  lastChecked: number;
+  lastWarnedDate: string;
+  latestVersion: string;
+}
+interface IVersionUpgradeWarningFrequency {
+  versionSyncDuration: number;
+}
```

Comment:
@naman-contentstack , can you pls move type def in interface folder ?

Reply:
Moved it to type def file, please check